File: src/Java/Heart Rate/HeartRate.java
public class HeartRate {
    public static void main(String args[]) {
        Kattio io = new Kattio(System.in);
        int cases = io.getInt();
        for (int i = 0; i < cases; i++) {
            double beats = io.getInt(), time = io.getDouble();
            double bpm = 60 * beats / time;
            double minabpm = bpm - 60 / time, maxabpm = bpm + 60 / time;
            io.println(minabpm + " " + bpm + " " + maxabpm);
        }
        io.flush();
    }
}

File: src/Python 3/Heart Rate/HeartRate.py
for i in range(int(input())):
    b, p = input().split()
    b = int(b)
    p = float(p)
    bpm = 60 * b / p
    minABPM = bpm - (60 / p)
    maxABPM = bpm + (60 / p)
    print(minABPM, bpm, maxABPM)